applets/clock/clock.c

```diff
@@ -1319,17 +1327,17 @@ clock_update_text_gravity (GtkWidget *label)
 }
 
 static inline void
-force_no_button_vertical_padding (GtkWidget *widget)
+force_no_button_padding (GtkWidget *widget)
 {
         GtkCssProvider  *provider;
 
         provider = gtk_css_provider_new ();
         gtk_css_provider_load_from_data (provider,
                                          "#clock-applet-button {\n"
-                                         "padding-top: 0px;\n"
-                                         "padding-bottom: 0px;\n"
-                                         "padding-left: 4px;\n"
-                                         "padding-right: 4px;\n"
+                                         "  padding-top: 0px;\n"
+                                         "  padding-bottom: 0px;\n"
+                                         "  padding-left: 0px;\n"
+                                         "  padding-right: 0px;\n"
```

Comment:
Why removing the padding for vertical panels?
I can't recall but there was a reason why we use the padding, many years ago.
For me it looks nicer with a padding. Using 2px seems OK for me.
Any way i don't want to block it.
But this needs to be moved in a single commit.
Otherwise you will never find the change in git history, when user will blame us about.

Reply:
And why adding the 2 blanks before padding-......?

Reply:
last time that i ask........